train data Mohamed Elmoudni

simonnyc opened the pull request:
Hi Prof, 
I just loaded csv file and 500 images starting Richard_Gere_0003.jpg to Rudolph_Giuliani_0021.jpg
Also my csv file user.id is: 1029
Thank you 
Mohamed

simonnyc committed:
Add files via upload

simonnyc committed:
Add files via upload

simonnyc committed:
Add files via upload

simonnyc committed:
Add files via upload

simonnyc committed:
Add files via upload

Hi Prof, 
This is Mohamed Elmoudni... I just loaded 500 images starting Richard_Gere_0003.jpg to Rudolph_Giuliani_0021.jpg

Thank you 
Mohamed

simonnyc committed:
Add files via upload

Hi Prof, 
This is Mohamed Elmoudni...this is an update to the legend csv file.. 
Thank you 
Mohamed

muxspace commented:
No white space in the CSV. Also, please update the legend.csv so it is free of conflicts.

simonnyc committed:
Add files via upload

no white space. Took the columns names as they existed already in the original file.

simonnyc committed:
Add files via upload

no white spaces.. I looked for conflicts in csv, I did not find any.

simonnyc commented:
Hi Prof, 
I took out the white spaces. The conflict i found is the 4 initial files which they are not there anymore.  Please let me know if it is ok now. 

Thank you 
Mohamed
